frontend/src/components/Explanation.js

```diff
@@ -1 +1,24 @@
-// This is where the explanation component will be created
\ No newline at end of file
+// This is where the explanation component will be created
+import React from 'react';
+function Explanation(){
+    return(
+        <div class = "jumbotron">
+            <h1 class = "display-4">
+                    Welcome!
+            </h1>
+            <p class = "lead">
+                The furry friend finder is a solution to help shelter animals in Dallas get adopted. It matches a person looking for a pet with a pet that is at an animal shelter. Take a look at a picture and description of a pet. Hit yes to pick a friend and no to skip!
```

Comment:
Miguel already addressed everything else. One minor change would be to capitalize 'Furry Friend Finder' since it is the name of our project/app